src/containers/org/MemberRolesContainer.js

```diff
@@ -23,6 +24,10 @@ import {
   paginationReducer,
 } from '../../utils/stateReducers/pagination';
 
+const Flex = styled.div`
+  display: flex;
+`;
```

Comment:
`Flex` isn't really doing anything. You can achieve the same result without the extra component by using a `div` directly.

```
<div>
  {
    pagedRoles.map(...)
  {
</div>
```

Additionally, this results in chips being right next to each other... a little space wouldn't hurt:

<img width="360" alt="Screen Shot 2021-02-08 at 3 11 24 PM" src="https://user-images.githubusercontent.com/440299/107293194-088eb500-6a20-11eb-9670-a9d52fa8025d.png">

Reply:
so I'm gonna keep the Flex and add a margin to the chips with it.